vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -3092,8 +3092,9 @@ public void setMultiSort(boolean multiSort) {
      *         {@code false} otherwise
      */
     public boolean isMultiSort() {
-        String multiSort = getElement().getAttribute("multi-sort");
-        return multiSort == null ? false : Boolean.valueOf(multiSort);
+        return Optional.ofNullable(getElement().getAttribute("multi-sort"))
```

Comment:
I think usage of stream API and Optional is excess work here, because we are not dealing with any stream and the only values that we deal with are `null` and `""`, so I think it is better to change it with basic `if ... else` statements.

Reply:
I'm not using the Stream API here. I'm using `Optional` to make the code `null` safe. Also to make an empty value of the `multi-sort` attribute the same as `true`.

The current change is equivalent to:
```Java
String multiSort = getElement().getAttribute("multi-sort");
if (multiSort != null && multiSort.length() == 0) {
    multiSort = "true";
}
return multiSort == null ? false : Boolean.valueOf(multiSort);
```

Reply:
IMO if we reduce it only to this code it works, unless I am missing something, no need to use `valueOf`:

String multiSort = getElement().getAttribute("multi-sort");
 return multiSort == null ? false : true;

Reply:
It does given the current implementation. However, it is assuming we will never change it to `multi-sort="false"` one day. I was just being extra cautious.

Reply:
I changed the implementation to not use `Optional` so it is a bit easier to read.